test.sh

```diff
@@ -1,28 +1,26 @@
 #!/usr/bin/env bash
 
 set -x
-set -e
+#set -e
 
-mkdir -p features/support
-cp _circleci_formatter.rb features/support
+FAILURE=false
+
+#mkdir -p features/support
+#cp _circleci_formatter.rb features/support
```

Comment:
Looks like you're doing this here: https://github.com/circleci/cucumber-formatter/pull/5/files#diff-1b531c5e405e0bc95890d52cf2f2f5b5L37